framework/graphics/vulkan_resources_util.cpp

```diff
@@ -1877,7 +1877,7 @@ void VulkanResourcesUtil::ReadFromImageResourceLinear(VkImage                ima
     subresource_sizes.clear();
 
     const double texel_size = vkuFormatTexelSizeWithAspect(format, aspect);
-    assert(texel_size == static_cast<uint64_t>(texel_size));
+    assert(texel_size == static_cast<double>(static_cast<uint64_t>(texel_size)));
```

Comment:
`GFXRECON_ASSERT` and.. how about using `floor(texel_size)`?

Reply:
damn me, I had a blackout and was searching for 'fract'

Reply:
changed to use `std::floor`